cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -74,7 +76,7 @@ def test_get_course_list(self):
         """
         Test getting courses with new access group format e.g. 'instructor_edx.course.run'
         """
-        course_location = self.store.make_course_key('Org1', 'Course1', 'Run1')
+        course_location = CourseLocator('Org1', 'Course1', 'Run1')
```

Comment:
Don't change this. Using the modulestore to construct the `CourseKey` is less fragile.